LICENSE

```diff
@@ -1,7 +1,6 @@
-
                                  Apache License
                            Version 2.0, January 2004
-                        https://www.apache.org/licenses/
+                        http://www.apache.org/licenses/
```

Comment:
I've filed opencontainers/project-template#25 to bring the template exactly in line with the canonical source.